.github/workflows/build-oca-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - "14.0*"
+  push:
+    branches:
+      - "14.0"
+      - "14.0-ocabot-*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
+
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: Dockerfile
+          tags: ghcr.io/oca/oca-custom:14.0-oca-latest
+      -
+        name: Test install oca_all in docker image
+        run: |
+          docker compose up --wait db kwkhtmltopdf
+          docker compose run --rm odoo odoo -c /etc/odoo.cfg -i oca_all --without-demo= --stop-after-init
+          # purpose it to test update entrypoints
+          docker compose up --wait odoo
```

Comment:
This is not strictly necessary and takes some time likes `-u all`. This test some how 000_10_odoo-update entry point with a database installed  (it not strictly assert that the update has been done). While the -i oca_all test without odoo database!

Any opinion on this?